fab/fabfile.py

```diff
@@ -679,6 +680,7 @@ def deploy():
 
 def _deploy_without_asking():
     try:
+        _tag_commit()
```

Comment:
This adds the tag at the start of the deploy - but I was thinking maybe this should instead go in `record_successful_deploy`